animate_path: false
animate_endeffector: true
animate_endeffector_segment: r_gripper_tool_frame
trajectory_duration: 5.0
trajectory_discretization: 0.05
max_iterations: 200
max_iterations_after_collision_free: 200
smoothness_cost_velocity: 0.0
smoothness_cost_acceleration: 1.0
smoothness_cost_jerk: 0.0
smoothness_cost_weight: 0.00000001
constraint_cost_weight: 0.2
#torque_cost_weight: 0.001

use_cumulative_costs: false
num_rollouts: 10
num_reused_rollouts: 5
#noise_stddev: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]
#noise_stddev: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
noise_stddev: [2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0]
#noise_stddev: [5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0]
#noise_stddev: [3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0]
#noise_decay: [0.995, 0.995, 0.995, 0.995, 0.995, 0.995, 0.995]
#noise_decay: [0.995, 0.995, 0.995, 0.995, 0.995, 0.995, 0.995]
noise_decay: [0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999]
#noise_decay: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
num_time_steps: 99
write_to_file: false
